coachmark/src/main/java/com/mercadolibre/android/andesui/coachmark/presenter/CoachmarkPresenter.kt

```diff
@@ -31,7 +31,7 @@ internal class CoachmarkPresenter(private val view: CoachmarkViewInterface) {
 
         stepReferenced.view.let {
```

Comment:
```suggestion
        stepReferenced.view?.let {
```

Puedes hacer la validación de view? antes para que el it siempre sea != null, y evitar usar it!!